fake-gcs-server.json

```diff
@@ -0,0 +1,6 @@
+{
+  "gcs_base_url": "http://localhost:4443",
+  "disable_oauth": true,
+  "client_email": "",
+  "private_key": ""
+}
```

Comment:
Minor:
It seems like better hygiene to move this little stub JSON cred into a "better" place. Any objections to e.g. `resources/`?

Reply:
Yeah, moving that to `resources/` seems like a good idea. Even better if this credential can be passed via env vars/temp file like it's [done in delta-rs](https://github.com/delta-io/delta-rs/blob/main/rust/src/test_utils.rs#L40-L47).

Btw, given that XML multipart upload mock API seems to be [coming soon](https://github.com/fsouza/fake-gcs-server/issues/852#issuecomment-1493331946) to fake-gcs-server, I'd leave all the test code in this PR (but only remove the paramatrization from `test_insert_two_different_schemas` so that the tests pass for now).

Reply:
Thanks for the link to the `delta-rs` approach - I like it, arguably it's a bit nicer/cleaner to handle it that way vs leaving a single purpose stub file hanging around the repo.

RE the XML multipart upload, thanks for sharing that! 🤞 it lands soon, will be great to tie the knot on this